server/lib/queries.js

```diff
@@ -310,6 +310,7 @@ export const usersToNotifyForUpdateSQLQuery = `
   ), member_collectives AS (
     SELECT mc.*
     FROM "Members" m
+    FULL OUTER JOIN "Collectives" c ON (c.id = :collectiveId OR c."ParentCollectiveId" = :collectiveId)
```

Comment:
Why do we need to re-include the collective (`c.id = :collectiveId`)?

And can't we update this query without a new join + condition simply by updating the `WITH collective AS (` to:

```sql
WITH collective AS (
    SELECT c.*
    FROM "Collectives" c
    WHERE id = :collectiveId OR "ParentCollectiveId" = :collectiveId
```